### PR TITLE
added editorconfig as seen in #167

### PR DESCRIPTION
**Pull request by OmegaRogue**

### CODE REVIEW

**Comment by OmegaRogue** [comment]
#167

**Comment by NicoHood** [comment]
I dont think this is required, as it is a config for your personal editor, that I or other people do not use.

**Comment by OmegaRogue** [comment]
editorconfig is cross editor https://editorconfig.org/

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[*]
+charset = utf-8
+end_of_line = crlf
```
Comment (NicoHood) [inline .editorconfig:3]: I guess this is wrong, isnt it? I'd prefer the unix linefeed. If some files are not yet in that format, that might be true, as they are very old.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# Crashlytics plugin (for Android Studio and IntelliJ)
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+fabric.properties
+
+# Editor-based Rest Client
+.idea/httpRequests
+
```
Comment (NicoHood) [inline .gitignore:55]: I think this one is a bit oversized, I dont want to add this "bloat" to every project. Correct me, if I am wrong.

**Comment by NicoHood** [comment]
I revisited this, and you are right. I totally missed the point with editor config. I have no much experience with this, but adding a basic config (with spaces!!!) is fine for me. Checkout my comments.

**Comment by NicoHood** [comment]
Fixed with https://github.com/NicoHood/HID/commit/b179743ba750d493c2bfbe892c6c0d7104943e6a

**Pull request closed without merging**